tests/eth.js

```diff
@@ -66,23 +66,7 @@ describe('ETH Tests', function() {
       currency,
       rawTx: '0x' + serializedTx.toString('hex')
     });
-    assert.isTrue(sentTx.from === txData.from.toLowerCase());
-    assert.isTrue(sentTx.to === txData.to.toLowerCase());
-    expect(sentTx).to.have.property('transactionHash');
-    expect(sentTx).to.have.property('transactionIndex');
-    expect(sentTx).to.have.property('blockHash');
-    expect(sentTx).to.have.property('blockNumber');
-    expect(sentTx).to.have.property('gasUsed');
-    expect(sentTx).to.have.property('from');
-    expect(sentTx).to.have.property('to');
-    expect(sentTx).to.have.property('cumulativeGasUsed');
-    expect(sentTx).to.have.property('contractAddress');
-    expect(sentTx).to.have.property('logs');
-    expect(sentTx).to.have.property('status');
-    expect(sentTx).to.have.property('logsBloom');
-    expect(sentTx).to.have.property('v');
-    expect(sentTx).to.have.property('r');
-    expect(sentTx).to.have.property('s');
+    expect(sentTx).to.equal('0x66ce93e97e48d09e1b5adac6b953e8faa2cd29ed2a8c5325c331648a6dc437f4');
```

Comment:
Probably want to do a length check since another test that sends a transaction being added could alter this txid

Reply:
added length check for 66